guides/gr-iapr.mdx

```diff
@@ -0,0 +1,275 @@
+---
+title: "Greece - myDATA & IAPR"
+description: Send invoices and credit notes to the Greek IAPR via the myDATA platform.
+---
+
+## Introduction
+
+The myDATA (My Digital Accounting and Tax Application) platform is Greece’s standardized electronic invoicing and bookkeeping system, managed by the Independent Authority for Public Revenue (IAPR). This system enables businesses to report invoices and accounting data in real time, ensuring compliance with Greek tax regulations.
+
+This guide will walk you through the process of reporting invoices to the IAPR using myDATA, first registering a supplier, and then issuing invoices and credit notes in their name.
+
+## Prerequisites
+
+To issue service notes in Greece, you will need:
+
+- Supplier details, including company name, TIN, and address.
+- Customer details for B2B transactions, including name, TIN or international ID, and address.
+- Line and item details, including quantity, price, description, applicable VAT rates, and, optionally, income classification codes.
+- To have chosen an invoice series.
+- Have authorized "ILYDA", the provider used by Invopop, to issue invoices on the supplier's behalf in the IAPR, and followed the specific on-boarding process (details below).
+
+## Supplier on-boarding
```

Comment:
Thanks for the input! In the TBAI or SAT guides, suppliers aren't registered in the "Setup" section, only the workflow to register suppliers is created there. In the SAT guide, the actual supplier registration is described in the "Running" section. For TBAI, the actual registration is not described at all. 

Since, eventually, Greece supplier registration will work as in Mexico, to be coherent with that, in https://github.com/invopop/docs.invopop.com/pull/26/commits/2988c3e2ace99b495d1c5f5bb6bb03d184fab45b, I moved this section to the "Running" section.